From 6f262eb7e4f7682a34b53d5c4328fee1a4d817de Mon Sep 17 00:00:00 2001
From: Hongbo Zheng <zhenghongbo3@huawei.com>
Date: Fri, 23 Apr 2021 17:56:34 +0800
Subject: [PATCH 136/189] net/hns3: disable MAC status report interrupt

Disable the MAC status report interrupt which hns3 driver not concern
currently.

Fixes: 5f8845f4ba8f ("net/hns3: process MAC interrupt")
Cc: stable@dpdk.org

Signed-off-by: Hongbo Zheng <zhenghongbo3@huawei.com>
Signed-off-by: Min Hu (Connor) <humin29@huawei.com>
---
 drivers/net/hns3/hns3_ethdev.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/net/hns3/hns3_ethdev.c b/drivers/net/hns3/hns3_ethdev.c
index 887c008..cea7926 100644
--- a/drivers/net/hns3/hns3_ethdev.c
+++ b/drivers/net/hns3/hns3_ethdev.c
@@ -4892,8 +4892,6 @@ hns3_update_link_status(struct hns3_hw *hw)
 	if (state != hw->mac.link_status) {
 		hw->mac.link_status = state;
 		hns3_warn(hw, "Link status change to %s!", state ? "up" : "down");
-		hns3_config_mac_tnl_int(hw,
-					state == ETH_LINK_UP ? true : false);
 		return true;
 	}
 
-- 
2.7.4

